src/docs/src/api/database/find.rst

```diff
@@ -1472,9 +1472,7 @@ it easier to take advantage of future improvements to query planning
                     "year",
                     "title"
                 ],
-                "r": [
-                    49
-                ],
+                "r": "1",
```

Comment:
why a string and not a number?

Reply:
For some reason, the value of `r` is stored as a string internally, and converted back to an integer whenever needed.  For examples, see [`fabric_doc_open`](https://github.com/apache/couchdb/blob/3e8155e6c3b4d9906c4d1ac24504470663d30b3f/src/fabric/src/fabric_doc_open.erl#L45), [`fabric_open_revs`](https://github.com/apache/couchdb/blob/3e8155e6c3b4d9906c4d1ac24504470663d30b3f/src/fabric/src/fabric_open_revs.erl#L87), and [`fabric_doc_open_revs`](https://github.com/apache/couchdb/blob/3e8155e6c3b4d9906c4d1ac24504470663d30b3f/src/fabric/src/fabric_doc_open_revs.erl#L40).  I did not want to change this, but I can do that if desired.

Reply:
hrm, bizarre. ok. keep as a string for now. cc @janl

Reply:
weird, I have no strong opinion on this. but we should consider mirroring what db info returns

Reply:
additionally: `r` can’t be anything else but 1, so is spelling this out maybe redundant at best and confusing at worst (if people think they can change it?)

Reply:
if db_info returns a number, agreed that we should return a number here. and, yes, we don't do quorum reads for indexes, the code simply doesn't exist.

Reply:
The value of `r` is mirrored back from the query parameters.  I understand that it has no effect, but if one submits a query to say 'r=100` it will be reflected in the `_explain` output (and passed around in the code).

Reply:
FWIW I didn't have any `r` parameter when I stumbled onto this so I guess it also reflects the default

Reply:
Yes, if the `r` query parameter is not set, its default value will be added.

Reply:
Mango does use quorum index reads: https://github.com/apache/couchdb/blob/main/src/mango/src/mango_cursor_view.erl#L659-L674